src/pull_automerge/core.clj

```diff
@@ -50,13 +49,24 @@
   (println (str "Get pull request " pull-number " status: " (result :status)))
   (parse-string (result :body) true))
 
+(defn contains-label [labels label]
+  (> (count (filter #(= (% :name) label) labels)) 0))
+
+(defn filter-by-label [ response-data label ]
+  (filter
+    #(contains-label (% :labels) label)
+    (json/read-str (response-data :body) :key-fn keyword)))
+
 (defn get-oldest-issue-as-pull-request [pulls org repo label options]
-  (if (= 0 (count pulls))
+  (def priority "High Priority")
+  (def pull (or 
+            (first (filter-by-label pulls priority))
+            (first (json/read-str (pulls :body) :key-fn keyword))))
```

Comment:
Just one tab in instead of lining it up. You could try bringing the `or` down a line with a tab-in and then the `first`s with one additional tab to see if that feels better.